subsys/trusted_storage/include/psa/error.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019-2021, Arm Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ */
```

Comment:
If somebody knows how we handle the copyright for copied files, I'm happy to get input.

Reply:
I suggest we put the PSA headers under the `ext/` folder.
This one is allowed to have different licenses, and it more accurately represents how we got these headers.

Reply:
Although it is not strictly forbidden to include 3rd-party headers and source code anywhere in the tree, I agree with @joerchan that having them in `ext/psa` or whatever other subfolder you choose promotes better structure and a clean division of ownership of the headers.